src/business-logic/services/test/rundown-timeline-service.spec.ts

```diff
@@ -55,6 +56,22 @@ describe(RundownTimelineService.name, () => {
 
       await expect(() => testee.deleteRundown(rundown.id)).rejects.toThrow(ActiveRundownException)
     })
+
+    it('throws an exception, when trying to active a rundown when there is another already activated rundown', async () => {
+      const mockRundownRepository: RundownRepository = mock<RundownRepository>()
+
+      const activeBasicRundown: Rundown = EntityMockFactory.createRundown({ isRundownActive: true })
+
+      const basicRundownArray: Rundown[] = [activeBasicRundown];
```

Comment:
Instead of naming it with what technical construct is used to hold the entitites, you can refer to the entities in plural such that this becomes: `const basicRundowns: Rundown = [activeBasicRundown]`.